### PR TITLE
Add drakeborgs

### PR DESCRIPTION
**Pull request by metalgearsloth**
image

why:  they're cute.

### CODE REVIEW

**Comment by LankLTE** [comment]
abusive maintainers closing PRs just because they don't like them... smh bro this game is dead... 1984...

**Comment by DogZeroX** [comment]
Finish what you started.

**Comment by mirrorcult** [comment]
>not even any code
smh

**Comment by Tunguso4ka** [comment]
tgstation was a mistake, port everything from vorestation /j

**Comment by LankLTE** [comment]
> not even any code
 
sounds like you need to write some

**Comment by Jackal298** [comment]
Pain
There are things you shouldn't remember

**Comment by Errant-4** [comment]
You may have thought you were making a joke when you opened this PR, but it was the universe itself making it's will manifest

**Comment by Anzarot121** [comment]
real

**Comment by moonheart08** [comment]
no

**Comment by mirrorcult** [comment]
now that this issue is closed no one will see this comment. im gay

File: Resources/gottem.txt
Comment (CaptainSqrBeard) [inline Resources/gottem.txt:1]: there should be a large ascii art of trollface

Comment (deltanedas) [reply Resources/gottem.txt:1]: :trollface:

**Comment by Jezithyr** [comment]
good pr. will merg 11/10

**Pull request closed without merging**